client/cli/src/lib.rs

```diff
@@ -79,26 +80,18 @@ const DEFAULT_KEYSTORE_CONFIG_PATH : &'static str =  "keystore";
 /// The maximum number of characters for a node name.
 const NODE_NAME_MAX_LENGTH: usize = 32;
 
-/// Executable version. Used to pass version information from the root crate.
-#[derive(Clone)]
-pub struct VersionInfo {
-	/// Implementaiton name.
-	pub name: &'static str,
-	/// Implementation version.
-	pub version: &'static str,
-	/// SCM Commit hash.
-	pub commit: &'static str,
-	/// Executable file name.
-	pub executable_name: &'static str,
-	/// Executable file description.
-	pub description: &'static str,
-	/// Executable file author.
-	pub author: &'static str,
-	/// Support URL.
-	pub support_url: &'static str,
-	/// Copyright starting year (x-current year)
-	pub copyright_start_year: i32,
-}
+#[cfg(test)]
+#[doc(hidden)]
+pub const TEST_VERSION_INFO: &'static VersionInfo = &VersionInfo {
```

Comment:
Why is that not defined in the tests mod below?

Reply:
I believe it can be used in other tests in other crates. I thought it would be a good idea. What do you think?

Reply:
(I plan to write some tests for the CLI)